site/_includes/partials/toc-side.njk

```diff
@@ -7,6 +7,8 @@
   still have the right margin. Otherwise the article will shift out of place.
 #}
 <div class="toc-container display-none xl:display-block pad-left-400 pad-right-400 {% if not tocContents %}visibility-hidden{% endif %}">
-  <div class="type--h6 color-secondary-text">{{ 'i18n.common.toc' | i18n(locale) }}</div>
-  <div class="toc__wrapper">{{ tocContents | safe }}</div>
+  <toc-active>
```

Comment:
hm... Ewa wrote a similar component for the Learn CSS course (web-scroll-spy)
https://github.com/GoogleChrome/web.dev/blob/main/src/lib/components/ScrollSpy/index.js

It seems ungood to have both

Reply:
Well, there you go.

I could merge them and move it to webdev-infra. I'd probably rather get this in then move them though.

Reply:
Yeah ideally we should have one component that does it. I'm fine not blocking this PR if you plan to do a follow up that merges and moves them to infra.